save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/cvsscalculator/ScoreCalculator.kt

```diff
@@ -91,13 +92,18 @@ private fun calculate(baseMetrics: BaseMetrics): Float {
 @Suppress(
     "FLOAT_IN_ACCURATE_CALCULATIONS",
     "MagicNumber",
+    "VARIABLE_HAS_PREFIX",
 )
-// https://www.first.org/cvss/v3.1/specification-document#Appendix-A---Floating-Point-Rounding
-private fun roundup(number: Float): Float {
-    val value = (number * 100_000).roundToInt()
-    return if (value % 10_000 == 0) {
-        value / 100_000f
-    } else {
-        (value / 10_000 + 1) / 10f
-    }
+private fun calculate(baseMetricsV2: BaseMetricsV2): Float {
```

Comment:
I really do not understand, why these calculate methods are here and not in corresponding BaseMetrics class (which are children of a some common Interface with a `calculate` API)

Reply:
Fixed